lib/parseformat.js

```diff
@@ -126,6 +129,12 @@ function parseFormat (dateString, options) {
   format = format.replace(regexHoursAmPm, 'h$1')
   // 05:30:20 ☛ HH:mm:ss
   format = format.replace(regexHoursWithLeadingZeroMinutesSeconds, 'HH:mm:ss')
+  // 5:30:20.222 ☛ H:mm:ss.SSS
+  format = format.replace(regexHoursMinutesSecondsMilliseconds, 'H:mm:ss.SSS')
+  // 5:30:20.222 ☛ H:mm:ss.SS
+  format = format.replace(regexHoursMinutesSecondsCentiSeconds, 'H:mm:ss.SS')
+  // 5:30:20.222 ☛ H:mm:ss.S
+  format = format.replace(regexHoursMinutesSecondsDeciSeconds, 'H:mm:ss.S')
```

Comment:
Sorry I’ve just seen this now. I think the two comments above are wrong, it should be `5:30:20.2 ☛ H:mm:ss.SS` and `5:30:20.2 ☛ H:mm:ss.S`, right?